### PR TITLE
Add Notes to assets

### PR DESCRIPTION
**Pull request by jSylvestre**
Equipment

### CODE REVIEW

**Comment by jSylvestre** [comment]
Issue ##516

**Comment by jSylvestre** [comment]
Icon on Key details page if notes exist

image

**Comment by jSylvestre** [comment]
Close #516

File: Keas.Mvc/ClientApp/components/Access/AccessEditValues.tsx [hunk 1]
```diff
@@ -74,7 +74,18 @@ export default class AccessEditValues extends React.Component<IProps, {}> {
...
                     <FormFeedback>Item name is required</FormFeedback>
-                </FormGroup>
+                </FormGroup>
+                <div className="form-group">
+                    <label>Notes</label>
+                    <textarea
+                        className="form-control"
+                        disabled={this.props.disableEditing}
+                        value={
+                            this.props.selectedAccess.notes ? this.props.selectedAccess.notes : ""
```
Comment (srkirkland) [inline Keas.Mvc/ClientApp/components/Access/AccessEditValues.tsx]: simpler: `this.props.selectedAccess.notes || ""`

**Pull request merged**